spec/unit/views/components/paginated_collection_spec.rb

```diff
@@ -178,7 +178,7 @@ def paginated_collection(*args)
       end
 
       it "should display proper message (including number and not hash)" do
-        expect(pagination.find_by_class('pagination_information').first.content).to eq "Displaying <b>all 2</b> posts"
+        expect(pagination.find_by_class('pagination-information').first.content).to eq "Displaying <b>all 2</b> posts"
```

Comment:
Line is too long. [118/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.